functions/model/tasks/getTasks.js

```diff
@@ -0,0 +1,29 @@
+const { getTasks } = require("./lib");
+const { getOrgUseAppKey } = require("../organization/lib");
+
+exports.getTasks = function(request, response) {
+    const orgDomain = request.body.data.orgDomain;
+    const teamId = request.body.data.teamId;
+    const sprintNumber = request.body.data.sprintNumber;
+    const status = 200;
+    const promise1 = getTasks(orgDomain, teamId, sprintNumber).then((task) => {
```

Comment:
The method name getTasks is used multiple times, it might confuse the runtime engine with the concept of overloading.

And also not a healthy practice, Please use different names, and if you see anywhere happening as such into our code. please fix them.